uint16 number_leds
